spec/features/cancel_spec.rb

```diff
@@ -1,80 +1,95 @@
-describe "Canceling a request" do
-  let!(:client_data) { FactoryGirl.create(:ncr_work_order, :with_approvers) }
-  let!(:proposal) { client_data.proposal }
-
-  it "shows a cancel link for the requester" do
+describe 'Canceling a request' do
+  it 'shows a cancel link for the requester' do
+    proposal = FactoryGirl.create(:proposal)
     login_as(proposal.requester)
+
     visit proposal_path(proposal)
-    expect(page).to have_content("Cancel my request")
+
+    expect(page).to have_content('Cancel my request')
   end
 
-  it "does not show a cancel link for non-requesters" do
+  it 'does not show a cancel link for non-requesters' do
+    proposal = FactoryGirl.create(:proposal, :with_approver)
     login_as(proposal.approvers.first)
+
     visit proposal_path(proposal)
-    expect(page).to_not have_content("Cancel my request")
+
+    expect(page).to_not have_content('Cancel my request')
   end
 
-  it "prompts the requester for a reason" do
+  it 'prompts the requester for a reason' do
+    proposal = FactoryGirl.create(:proposal)
     login_as(proposal.requester)
+
     visit proposal_path(proposal)
     click_on('Cancel my request')
+
     expect(current_path).to eq("/proposals/#{proposal.id}/cancel_form")
   end
 
   context 'email' do
-    # TODO this shouldn't actually be needed
-    around(:each) { ActionMailer::Base.deliveries.clear }
-
-    it "send emails when cancellation is complete" do
+    it 'send emails when cancellation is complete' do
+      ActionMailer::Base.deliveries.clear
+      proposal = FactoryGirl.create(:proposal)
       expect(deliveries.length).to eq(0)
 
       login_as(proposal.requester)
       visit proposal_path(proposal)
       click_on('Cancel my request')
-      fill_in "reason_input", with: "This is a good reason for the cancellation."
+      fill_in 'reason_input', with: 'This is a good reason for the cancellation.'
       click_on('Yes, cancel this request')
 
-      expect(deliveries.length).to eq(2)
+      expect(deliveries.length).to eq(1)
```

Comment:
discovered why there was just 1 email:

When I updated these specs, I removed the common proposal record. Formerly, the proposal always had an approver but I realized we didn't need one for every spec.

In this case, I assumed we didn't need one because the approver was not mentioned so I assumed we didn't need one. But because of @phirefly 's comment about "the pending approver" receiving an email, I can see that the test was written with the expectation of a proposal with an approver.

When I added an approver, there were actually 3 emails sent. Upon further inspection with @pkarman , it was discovered that a cancellation email goes to the requester and to the approver. The third email is a comment email that goes to the approver as well. 

There is a story already for removing the comment email upon cancellation: https://www.pivotaltracker.com/n/projects/1149728/stories/99273060

For now, I am going to keep the expectation in the spec that there are 3 emails, because that is the current behavior, and I will tackle the "remove comment email" feature next (and update this spec accordingly)